solc/CommandLineInterface.cpp

```diff
@@ -105,6 +106,7 @@ static string const g_stdinFileNameStr = "<stdin>";
 static string const g_strMetadataLiteral = "metadata-literal";
 static string const g_strAllowPaths = "allow-paths";
 static string const g_strStandardJSON = "standard-json";
+static string const g_strCompactJSON = "compact-format";
```

Comment:
oh..that was't obvious because the order was messed a bit already....I will adjust the other ones too